Homework_7/7_2_ex/7_2_ex/tests.cpp

```diff
@@ -0,0 +1,54 @@
+#include "tests.h"
+#include "tree.h"
+#include "workingWithData.h"
+#include <string>
+#include <fstream>
+#include <cassert>
+
+using namespace std;
+
+bool getLineFromFile(istream& file, string& data)
+{
+	if (!getline(file, data))
+	{
+		return false;
+	}
+	if (data == "\0")
+	{
+		getline(file, data);
+	}
+	return true;
+}
+
+void tests()
+{
+	ifstream testDataFile("testData.txt", ios::in);
+	assert(testDataFile);
+	ifstream testAnswerFile("testAnswer.txt", ios::in);
+	assert(testAnswerFile);
+
+	string data = "";
+	string answer = "";
+
+	while (!testDataFile.eof())
+	{
+		if (!getLineFromFile(testDataFile, data))
+		{
+			break;
+		}
+
+		Tree* testTree = createTree();
+		makeExpressionTree(testTree, data);
+
+		testAnswerFile >> answer;
+
+		assert(countExpression(testTree) == atoi(answer.c_str()));
```

Comment:
```suggestion
		assert(countExpression(testTree) == stoi(answer));
```
atoi работает с сишными строками (поэтому c_str()), stoi --- с C++-ными